hack/build/etcd-backup-operator/build

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+function go_build {
+	echo "building "${1}"..."
+	if [ ! -z ${GOINSTALL+x} ] && [ "${GOINSTALL}" = "y" ]
+	then
+  		GOBIN=${bin_dir} GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go install -installsuffix cgo  -ldflags "$go_ldflags" ./cmd/${1}/
+		mv ${bin_dir}/${1} ${bin_dir}/etcd-backup-${1}
+	else
+  		GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o ${bin_dir}/etcd-${1} -installsuffix cgo  -ldflags "$go_ldflags" ./cmd/${1}/
+	fi
+}
+
+if ! which go > /dev/null; then
+	echo "golang needs to be installed"
+	exit 1
+fi
+
+GIT_SHA=`git rev-parse --short HEAD || echo "GitNotFound"`
+
+gitHash="github.com/coreos/etcd-operator/version/etcd-backup-operator.GitSHA=${GIT_SHA}"
+
+go_ldflags="-X ${gitHash}"
+
+bin_dir="$(pwd)/_backup_output/bin"
```

Comment:
put this on top of go_build func

Reply:
this is following the style of etcd operator build script

https://github.com/coreos/etcd-operator/blob/master/hack/build/operator/build#L31-L32

Reply:
> this is following the style of etcd operator build script

This is not a good reasoning. It is bad to put such global var here because it's used in other funcs too.